openerp/models.py

```diff
@@ -2896,6 +2896,7 @@ def _inherits_reload(cls):
         cls._inherit_fields = struct = {}
         for parent_model, parent_field in cls._inherits.iteritems():
             parent = cls.pool[parent_model]
+            parent._inherits_reload()
```

Comment:
If I understood how models are loaded, after this change, `_inherits_reload` could be called several times for the same model, if it happens that a parent model is loaded before a child model.
My first attempt was:
```python
if not parent._inherit_fields:
    parent._inherits_reload()
```
But I am not sure that's safe.